### ISSUE TITLE
Change default title `weight` from `bold` to `regular`

### ISSUE THREAD

**Issue by Andrew15-5** [Pablo-Gonzalez-Calderon/showybox-package#19]
I think that the default value should be "not bold", because I was trying to make `[*bold tittle*]` and it didn't work since the `weight` is already `"bold"`. I think that the user can either change the `weight` to the desired value or use Typst's built-in syntax for bold text.

**Comment by Pablo-Gonzalez-Calderon** [comment]
I think it's preferrable to have ``"bold"`` as default, because most of the users would like the title to be emphasized. Here's an example of when the title is ``"bold``" and ``"regular"``:
![image](https://github.com/Pablo-Gonzalez-Calderon/showybox-package/assets/30756770/4b07dc2c-e080-4ee9-891c-7e64621b5741)

In my opinion, the first title looks "more emphasized" than the second one. This way, users can just write a string as a title, and it'll be automatically set as bold. But this conception varies from person to person. As it's something quite subjective, I think the best is to keep this as an open discussion to hear other users opinion about this and get a consensus in the meantime.

**Comment by Andrew15-5** [comment]
Emphasized is a bad word, just because Typst's `emph` (emphasis) means italicize. But sure.

Even though I agree that in theory bold by default is a good idea, if we are speaking purely about "how easy it is to change to bold/regular", then bold by default isn't the best choice.

If we have bold by default, and we want to reset it to normal, we would have to do:

```typ
#showybox(
  title: "Title",
  title-style: (weight: "regular"),
  ...
)
```

If we have regular by default, and we want to set it to bold, we would have to do:

```typ
#showybox(
  title: strong("Title"),
  // or
  title: [*Title*],
  ...
)
```

We then remove repetitive stuff, and we can visually see which "convertion" is easier:

```
""
title-style: (weight: "regular"),

strong("")
// or
[**]
```

**Comment by Pablo-Gonzalez-Calderon** [comment]
Yeah, from that point of view it's easier to have `"regular"` as default 🤔

Also, I've checked ``tcolorbox`` manual, and saw that also was ``"regular"`` by default. So, in order to keep coherency, that should be showybox's case.

This discussion seems to be won 😅. I'll keep open this discussion anyways, but in the next version I'll set the default value to `"regular"` as the arguments seem enough.

**Comment by Andrew15-5** [comment]
Yay!)) Well, this was easy, thanks to the `tcolorbox`.
